test/model/renewal/test_model.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+
+from gluonts.model.renewal import DeepRenewalProcessEstimator
+from gluonts.mx.trainer import Trainer
+
+
+@pytest.fixture()
+def hyperparameters():
+    return dict(
+        num_cells=10,
+        num_layers=3,
+        context_length=10,
+        epochs=1,
+    )
+
+
+@pytest.mark.parametrize("hybridize", [True, False])
+def test_repr(repr_test, hyperparameters, hybridize):
+    hyperparameters.update(hybridize=hybridize)
+    repr_test(DeepRenewalProcessEstimator, hyperparameters)
+
+
+@pytest.mark.parametrize("hybridize", [True, False])
+def test_serialize(serialize_test, hyperparameters, hybridize):
+    hyperparameters.update(hybridize=hybridize)
+    serialize_test(DeepRenewalProcessEstimator, hyperparameters)
```

Comment:
I think none of these tests actually trains the model, right? Maybe we should have that, like for other models, as a smoke tests to ensure the thing runs